smart-contracts/contracts/mixins/MixinTransfer.sol

```diff
@@ -0,0 +1,183 @@
+pragma solidity 0.4.25;
+
+import './MixinDisableAndDestroy.sol';
+import './MixinApproval.sol';
+import './MixinKeys.sol';
+import './MixinLockCore.sol';
+import 'openzeppelin-solidity/contracts/utils/Address.sol';
+import 'openzeppelin-solidity/contracts/token/ERC721/IERC721Receiver.sol';
+import 'openzeppelin-solidity/contracts/math/SafeMath.sol';
+
+/**
+ * @title Mixin for the safe transfer related functions needed to meet the ERC721
```

Comment:
Pretty minor, but `for transfer related` since it includes more than just safe

Reply:
done.